dist/locales/en-AU.json

```diff
@@ -128,20 +128,20 @@
   "impac.widget.settings.limit_entries.display": "Display",
   "impac.widget.settings.limit_entries.entries": "entries",
 
-  "impac.dashboard_selector.delete_widget.description": "Are you sure you want to delete this analytics dashboard?",
+  "impac.dashboard_selector.delete_widget.description": "Are you sure you want to delete this {name}?",
   "impac.dashboard_selector.bootstrap_tabs.add_widget": "Add Widget",
   "impac.dashboard_selector.bootstrap_tabs.source": "Source:",
-  "impac.dashboard_selector.delete.delete_dashboard": "Delete Dashboard",
+  "impac.dashboard_selector.delete.delete_dashboard": "Delete {name}",
   "impac.dashboard_selector.delete.delete": "Delete",
   "impac.dashboard_selector.delete.cancel": "@:impac.common.action.cancel",
   "impac.dashboard_selector.dropdown.source": "Source:",
   "impac.dashboard_selector.dropdown.tooltip.change_name": "Change name",
-  "impac.dashboard_selector.dropdown.create_dashboard": "Create Dashboard",
-  "impac.dashboard_selector.dropdown.change_dashboard_name": "Change dashboard name:",
+  "impac.dashboard_selector.dropdown.create_dashboard": "Create {name}",
+  "impac.dashboard_selector.dropdown.change_dashboard_name": "Change {name} name:",
   "impac.dashboard_selector.dropdown.cancel": "@:impac.common.action.cancel",
   "impac.dashboard_selector.dropdown.confirm": "Confirm",
   "impac.dashboard_selector.dropdown.add_widget": "@:impac.dashboard_selector.bootstrap_tabs.add_widget",
-  "impac.dashboard_selector.dropdown.tooltip": "Delete Dashboard",
+  "impac.dashboard_selector.dropdown.tooltip.delete": "Delete {name}",
 
   "impac.widget.common.top_buttons.delete.title": "Are you sure you want to delete this widget ?",
   "impac.widget.common.top_buttons.delete.explanation": "(it will not erase your data)",
```

Comment:
Ok, just wasn't sure on the wording for this one, so maybe thought even when creating a Template, it made some sense to say dashboard here. But yep ok